packages/languages/src/node/language-server-contribution.ts

```diff
@@ -42,6 +42,10 @@ export abstract class BaseLanguageServerContribution implements LanguageServerCo
     @inject(ProcessManager)
     protected readonly processManager: ProcessManager;
 
+    constructor(
+        @inject(ILogger) @named('languages') protected logger: ILogger,
```

Comment:
Could you please replace it with property injection. Just to make sure we are not breaking anything due to the API changes.